lib/unsanitized.js

```diff
@@ -0,0 +1,83 @@
+// names of escaping functions that we acknowledge
+const VALID_ESCAPERS = ["Sanitizer.escapeHTML", "escapeHTML"];
+const VALID_UNWRAPPERS = ["Sanitizer.unwrapSafeHTML", "unwrapSafeHTML"];
+
+class Unsanitized {
```

Comment:
Sorry to start the bike shed 😭

The name `Unsanitized` does not speak for itself, I think `allowedExpression` is a fine method on its own, what do we gain from declaring this as a class? Also, what do we get from creating a const that has the `allowedExpression` function, that we wouldn't get otherwise? Is there some performance magic I'm not aware of?

If this lives in its own files (which, I agree it has with the split), I'd lean towards something more generic like `utils.allowedExpxpression` as its own function, without any kind of state (no class needed imho) :)